config/locales/client.en.yml

```
@@ -4054,6 +4054,7 @@ en:
       no_likes_body: "A great way to jump in and start contributing is to start reading conversations that have already taken place, and select the %{heartIcon} on posts that you like!"
       no_likes_others: "No liked posts."
       no_topics_title: "You have not started any topics yet"
+      no_topics_body: "It’s always best to <a href='%{searchUrl}'>search</a> for existing topics of conversation before starting a new one, but if you’re confident the topic you want isn’t out there already, go ahead and start a new topic of your very own. Look for the <kbd>+ New Topic</kbd> button at the top right of the topic list, category, or tag to begin creating a new topic in that area."
```

```suggestion
      no_topics_body: "It's best to <a href='%{searchUrl}'>search</a> for existing topics but you can always start a brand new topic using either the <kbd>+ New Topic</kbd> button in topic lists or by pressing <kbd>c</kbd> on your keyboard."
```

Just a shorter suggestion.

---

We agreed internally on the longest version for now.